source/common/stream_info/stream_info_impl.h

```diff
@@ -174,12 +174,20 @@ struct StreamInfoImpl : public StreamInfo {
 
   absl::optional<uint32_t> responseCode() const override { return response_code_; }
 
+  absl::optional<Grpc::Status::GrpcStatus> localReplyGrpcStatus() const override {
+    return local_reply_grpc_status_;
+  }
+
   const absl::optional<std::string>& responseCodeDetails() const override {
     return response_code_details_;
   }
 
   void setResponseCode(uint32_t code) override { response_code_ = code; }
 
+  void setLocalReplyGrpcStatus(Grpc::Status::GrpcStatus status) override {
```

Comment:
I don't see how it is called.

Reply:
Done. It will be called by FilterManager::sendLocalReply.